docs/source-fabric/fundamentals/launch.rst

```diff
@@ -15,11 +15,11 @@ To run your code distributed across many devices and many machines, you need to
 Simple Launch
 *************
 
-.. raw:: html
```

Comment:
This changes the format drastically. The video is no longer centered and much smaller, and controls are visible. See the original page and compare the two.

Reply:
ok, there is not much we can do about the center unless define in the template's CSS class... this the generated HTML source:
```html
<video autoplay="True" controls="True" loop="True" muted="True" preload="auto" width="500"><source src="[../_static/fetched-s3-assets/launch.mp4](https://github.com/Lightning-AI/lightning/pull/_static/fetched-s3-assets/launch.mp4)" type="video/mp4"></video>
```

Reply:
so it seems your original HTML was wrong as it ignored `max-width="500px"`
about controls, seems that `.. video::` has it on by default and disable need to be explicitly set by `:nocontrols:`

Reply:
tbh, I would always rather show the controls so the user can pause or better navigate it...